modules/pipeline/initialize.go

```diff
@@ -194,6 +201,9 @@ func do() (*httpserver.Server, error) {
 		return nil, err
 	}
 
+	// register cluster hook
+	registerClusterHook(bdl)
```

Comment:
Put registerClusterHook into pkg clusterinfo is better?
Put these two methods together at least.

Reply:
Or only one method here such as `doClusterAbout`.

Reply:
Move `registerClusterHook` into `clusterinfo.Initialize`